public/js/services/todo.service.js

```diff
@@ -1,4 +1,4 @@
-angular.module('todo.services', [])
+angular.module('todo.service', [])
```

Comment:
You don't have to create own module per each service. Usually you can create one module per feature. For our 'todo' app, we have only one feature - todo. One 'todo' module is enough